### ISSUE TITLE
Peer feedback

### ISSUE THREAD

**Issue by Joooooost** [NumEconCopenhagen/projects-2020-hugos-yndlings#1]
**1. The best part of the project was:**

- the plotly graph is really neat and the menu to control the graph is really nice 
- the background of what you are doing is explained good
- structure creates a good overview on the project
- the idea for the project is a good one

**2. The hardest part of the project to understand was:**

- the code for your graph lacked a little bit of comments

**3. This part of the project could be better documented:**

- create a markdown with all packages you need to install the code to install (e.g. Alpha Vantage, Plotly etc.) so it is straightforward to install before running the code

**4. An idea for an improvement/clarification could be:**

- don't write a function without inputs (like the one for the tickers) if it's just to download the data and you can't filter anyway
- don't have cells in noone else is supposed to run anyway or just comment it out so there is no risk of just running the whole code (and also highlight the cell with the creation of local data as it's not working without the previous step)
- you reference your own harddrive, so the users are required to adjust the path to the uploaded file with all the local data themselves
- use the yahoo finance API as it's easier to get the required information you want from there and you don't need to go the long way with download minutewise
- choose a date to drop the rows you don't want to show in the graph to have it more accurate
- the currency for the closure value is missing
- adjust the height of the graph as the absolute prices for these stocks are divering so you can better read the values from the cheaper stocks
- the last row of your dataframe is an empty row for the year 2006, it doesn't change the line in the end of the graph as its empty anyway but the xaxis is displying 2006 after 2020 (it is very confusing that the value is there), so maybe apply a procedure for the dataframe to drop all NaN values
- summing up over the monthly returns (percentage) for six months is not the same as having the difference between the value from six months earlier and today, so rather use the closing price six months from now vs. today to have the right picture of the momentum
- the .loc function with the date to find the values of the big five tech companies is not working as you wrote the date not in line with the date format in the dataframe (wrong order of date and seconds for hours are missing)

**5. An idea for an extension could be:**
- create widgets a range of times for the stock market to just show a specific part
- maybe load all data and let the user decide the share they want to see
- put your key fundamental data for the stock below the graph so you can read it as an output
